themes/jb/layouts/episode/single.html

```diff
@@ -71,20 +71,37 @@ <h2 class="has-text-centered-mobile">Guests</h2>
             {{ end }}
             </div>
           </div>
+          <!-- Sponsors -->
+          <div class="tile is-child columns is-flex-grow-0 is-hidden-tablet">
+            <div class="column px-0">
+              {{ with .Params.sponsors }}
+                <h2>Sponsors</h2>
+                <section class="section px-0 sponsor-list">
+                  {{ range . }}
+                    {{ $sponsor := index (where (where site.Pages "Section" "sponsors") "Params.shortname" .) 0 }}
+                    {{ if $sponsor }}
+                      {{ partial "sponsor/small.html" $sponsor }}
+                      <br />
+                    {{ end }}
+                  {{ end }}
+                </section>
+              {{ end }}
+            </div>
+          </div>
```

Comment:
Interesting approach! I was trying to figure out a way to change the order of the `div`s based on view port size. But I could not get that to work since they were in their own columns. I think this solution is honestly the simplest for what we are trying to accomplish. Maybe just throw a comment at each sponsor section to say why we have it defined twice in the HTML.

Reply:
Comment added. Yeah without using something like css grid or JS its not really possible.